app/controllers/static_pages_controller.rb

```diff
@@ -1,6 +1,5 @@
 class StaticPagesController < ApplicationController
   def home
-    @navbar = false
     @is_home_page = true
```

Comment:
I am not sure. The value of that instance variable decides whether to show `AdminFlashes`. But I have never come across the admin flashes. How are they actually used in the application?